docs/content/custom-replication-system.md

```diff
@@ -90,20 +95,18 @@ public class TransformSendSystem : CustomSpatialOSSendSystem<SpatialOSTransform>
 }
 ```
 
-> **Note**: The update objects are generated types from the [SpatialOS C# SDK](https://docs.improbable.io/reference/latest/csharpsdk/introduction).
+> **Note**: You need to create the `SchemaComponentUpdate` object with the correct component id. We provide Serialization methods to add data to this object automatically.
 
-Here's an example custom replication system for a component called `CubeColor`. The component has one event called `change_color` of the type `ColorData`.
+Here's an example custom replication system for a component called `CubeColor`. The component has one event called `change_color` of the type `ColorData` and has no fields.
 
 ```csharp
-[UpdateInGroup(typeof(UpdateGroupSpatialOSSend))]
-[UpdateBefore(typeof(UnityEngine.Experimental.PlayerLoop.FixedUpdate))]
 public class CubeColorSendSystem : CustomSpatialOSSendSystem<SpatialOSCubeColor>
 {
     public struct ColorData
     {
         public readonly int Length;
         public ComponentDataArray<SpatialOSCubeColor> CubeColors;
-        public ComponentDataArray<EventSender<SpatialOSCubeColor>> CubeColorEventSenders;
+        public ComponentDataArray<CuberColor.EventSenders.ChangeColor> ChangeColorEventSenders;
```

Comment:
in our code its `CuberColor.EventSender.ChangeColor` (without the s at the end of  Eventsenders)